jobOptions.mac

```diff
@@ -15,4 +15,17 @@
 /WCSim/physics/list WCSim # included for backward-compatibility.  Requires jobOptions2.mac
 
 
-
+#choose the Trigger & Digitizer type (and options)
+# set at least one (both to be sure) to SKI_SKDETSIM if you want to use the old combined Digitization & Trigger classes
+/DAQ/Digitizer SKI
+/DAQ/Trigger NHits
+#do we want to save only triggered events (mode 0), both triggered events & failed events (mode 1), or only failed events (mode 2)?
+/DAQ/TriggerSaveFailures/Mode 0
+#what trigger time to use for the failed events?
```

Comment:
There are still some questions in the comments here regarding the electronics settings. Is there a plan to ask an expert about these numbers?